images-list

```diff
@@ -1,14 +1,10 @@
-istio/sidecar_injector rancher/istio-sidecar_injector 1.5.9
-istio/galley rancher/istio-galley 1.5.9
-istio/mixer rancher/istio-mixer 1.5.9
-istio/pilot rancher/istio-pilot 1.5.9
-istio/citadel rancher/istio-citadel 1.5.9
-istio/kubectl rancher/istio-kubectl 1.5.9
```

Comment:
Is there no kubectl in the manifests ?

Reply:
From what I saw in the manifests, kubectl was only used in the addons, specifically for grafana dashboard.